internal/rbd/rbd_util.go

```diff
@@ -422,6 +426,28 @@ func createImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 		}
 	}
 
+	if pOpts.StripCount != 0 {
+		logMsg += fmt.Sprintf(", strip count %d, strip unit %d", pOpts.StripCount, pOpts.StripUnit)
+		err := options.SetUint64(librbd.RbdImageOptionStripeCount, pOpts.StripCount)
+		if err != nil {
+			return fmt.Errorf("failed to set strip count: %w", err)
+		}
+		err = options.SetUint64(librbd.RbdImageOptionStripeUnit, pOpts.StripUnit)
+		if err != nil {
+			return fmt.Errorf("failed to set strip unit: %w", err)
+		}
+	}
+
+	if pOpts.ObjectSize != 0 {
+		logMsg += fmt.Sprintf(", object size %d", pOpts.ObjectSize)
+		err := options.SetUint64(librbd.RbdImageOptionOrder, uint64(pOpts.ObjectSize))
```

Comment:
This is wrong: `RbdImageOptionOrder` expects a log-base-2 of `ObjectSize`, e.g. 22 for 4M.